src/cf.data.pre

```diff
@@ -2570,7 +2570,15 @@ DOC_START
 	Allows you to apply a Netfilter mark value to packets being transmitted
 	on the client-side, based on an ACL.
 
-	clientside_mark mark-value [!]aclname ...
+	clientside_mark mark[/mask] [!]aclname ...
+
+	Mark and mask are unsigned integers (hex, octal, or decimal).
+	Mask can be used if a mark set by other agents before squid (e.g., iptables)
```

Comment:
I would rephrase this sentence to polish and clarify: `Use the mask to preserve the mark previously set by other agents (e.g., iptables)` or `The mask may be used to preserve the mark previously set by other agents (e.g., iptables)`. The use of the mask is conditional on admin needs/desires, not on the "other agents" actions.